router-api/openapi.json

```diff
@@ -0,0 +1,534 @@
+{
+    "openapi": "3.0.0",
+    "info": {
+      "title": "GlueX Router API",
+      "description": "The GlueX Router API provides advanced routing for seamless trading across decentralized finance (DeFi). Utilizing multihop routing, it identifies the best prices and optimal trading paths, ensuring users achieve the most efficient and cost-effective trades. The API empowers developers with flexibility and performance to integrate powerful trading solutions into their applications.",
+      "version": "1.0.0"
+    },
+    "servers": [
+      {
+        "url": "https://router.gluex.xyz",
+        "description": "Production server"
+      }
+    ],
+    "paths": {
+      "/price": {
+        "get": {
+          "summary": "Get trade price details",
```

Comment:
Let's refrain from using the term "trade". The router is meant for any type of DeFi interaction.